File: console/android/ORLib/src/main/java/io/openremote/orlib/models/ORConsoleConfig.kt
package io.openremote.orlib.models

data class ORConsoleConfig (
    val showAppTextInput: Boolean,
    val showRealmTextInput: Boolean,
    val app: String?,
    val allowedApps: List<String>?,
    val apps: Map<String, ORAppInfo>?
)

File: console/iOS/ORLib/ORLib/Models/ORConsoleConfig.swift
//
//  ORConsoleConfig.swift
//  ORLib
//
//  Created by Eric Bariaux on 21/07/2022.
//

import Foundation

public struct ORConsoleConfig: Codable {
    
    public init() {
    }
    
    public var showAppTextInput = false
    public var showRealmTextInput = true
    public var app: String?
    public var allowedApps: [String]?
    public var apps: [String:ORAppInfo]?
}
